Automattic/newspack-blocks
# Add Spanish translations (with a focus on the donation block)

### All Submissions:


### Changes proposed in this Pull Request:

Through some trial and error, this PR translates the Donations block into Spanish.

As part of the work, I needed to adjust some of the strings in the block so they were identical on the front-end and back end (which simplified the POT file).

Closes #75.

**Edited to add:** I largely followed the process outlined [in this post](https://getwithgutenberg.com/2019/04/how-to-internationalize-your-block/), including renaming the translation files for JSON, to get this to work in the editor.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build:webpack`.
2. Navigate to Settings > General, and switch the site language to Spanish.
3. Add a Donate block to the editor; confirm that the content on all three tabs, and when you click 'Other', is translated into Spanish.
4. Save and publish the block; confirm that the output on the front-end is translated into Spanish. 

### Other information:
